backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImpl.java

```diff
@@ -338,4 +338,9 @@ public String setExternalId(String externalId)
 	{
 		return this.externalId = externalId;
 	}
+
+	public void setSalesRep_ID(final UserId userId)
+	{
+		this.salesRepId = userId;
+	}
```

Comment:
Why? The field is already annotated with `@Setter`

Reply:
correction done